spyderlib/plugins/help.py

```diff
@@ -185,7 +185,7 @@ def __init__(self, parent):
 
         self.webview = FrameWebView(self)
         self.find_widget = FindReplace(self)
-        self.find_widget.set_editor(self.webview)
+        self.find_widget.set_editor(self.webview._webview)

```

Comment:
Using a private variable is not good practice. Maybe just add a property to `self.webview` to access `_webview` ? This way no other code has to change.